guides/common/modules/con_provisioning-virtual-machines-in-proxmox.adoc

```diff
@@ -1,7 +1,7 @@
 [id="Provisioning_Virtual_Machines_in_Proxmox_{context}"]
 = Provisioning virtual machines in Proxmox
 
-Proxmox is open source software made for virtualization using KVM hypervisor and LXC containers.
+Proxmox is open source software made for virtualization by using KVM hypervisor and LXC containers.
```

Comment:
This sounds strange. You might want to break it into 2 sentences, based on the [Proxmox site description](https://www.proxmox.com/en/proxmox-virtual-environment/overview):

> Proxmox Virtual Environment is a complete, open-source server management platform for enterprise virtualization. It tightly integrates the KVM hypervisor and Linux Containers (LXC), software-defined storage and networking functionality, on a single platform. With the integrated web-based user interface you can manage VMs and containers, high availability for clusters, or the integrated disaster recovery tools with ease.

Reply:
```suggestion
Proxmox Virtual Environment is a complete, open-source server management platform for enterprise virtualization.
It tightly integrates the KVM hypervisor and Linux Containers (LXC), software-defined storage and networking functionality.
```
Let me be a happy copy :cat: